src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -76,16 +76,85 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
         approved: true
       }
     });
-  }
+    const element = await prisma.WBS_Element.findUnique({
+      where: {wbsElementId: foundCR.wbsElementId}
+    });
 
+    
+    if(!element.workPackage)
+    {
+      if(element.project != null)
+      {
+        const updatedBudgetImpact = element.project.budgetImpact + foundPs.budgetImpact;
+        const change = {
+          changeRequestId: crId,
+          implementerId: reviewerId,
+          detail: 'Budget Impact: '.concat(element.project.budgetImpact.toString, '->', updatedBudgetImpact.toString)
```

Comment:
we recently merged in a helper function to generate these strings for you (buildChangeDetail)